src/main/java/com/exactprosystems/remotehand/windows/actions/SwitchActiveWindow.java

```diff
@@ -36,14 +38,26 @@
 public class SwitchActiveWindow extends WindowsAction {
 
 	private static final String WINDOW_NAME_PARAM = "windowname",
-			ACCESSIBILITY_ID_PARAM = "accessibilityid";
+			ACCESSIBILITY_ID_PARAM = "accessibilityid",
+			MAX_TIMEOUT_PARAM = "maxtimeout";
 	private static final String HANDLE_ATTRIBUTE = "NativeWindowHandle";
 
 	private static final Logger loggerInstance = LoggerFactory.getLogger(SwitchActiveWindow.class);
 
 	@Override
 	public String run(WindowsDriverWrapper driverWrapper, Map<String, String> params, WindowsSessionContext.CachedWebElements cachedWebElements) throws ScriptExecuteException {
 
+		// Action is best usage for login windows, where login window runs under the same process that 
```

Comment:
replace the comment with:
// Action is useful for login windows, where login window belongs to the same process as
// main application. Action is able to catch required window even if there are multiple windows opened
// that satisfy chosen criteria.